examples/hacspec-poly1305/src/poly1305.rs

```diff
@@ -1,61 +1,75 @@
 // Import hacspec and all needed definitions.
 use hacspec_lib::*;
 
+// Type definitions for use in poly1305.
 bytes!(KeyPoly, 32);
 
 const BLOCKSIZE: usize = 16;
-// Type definitions for use in poly1305.
 
 // These are type aliases for convenience
-bytes!(Block, BLOCKSIZE);
+array!(Block, BLOCKSIZE, U8);
```

Comment:
Why not `bytes!`? If we really can't use it, we should make sure we know why and remove it from the library.

Reply:
Oups sorry, a leftover of some macro debugging, switching it to `bytes!` as it should be :)

Reply:
does bytes! always mean secret bytes?

Reply:
`bytes!` is always secret bytes, you can use `public_bytes!` otherwise.